internal/locate/pd_codec.go

```diff
@@ -71,7 +72,7 @@ func NewCodecPDClientWithKeyspace(mode apicodec.Mode, client pd.Client, keyspace
 		return nil, err
 	}
 
-	return &CodecPDClient{client, codec}, nil
+	return &CodecPDClient{client.WithCallerComponent("codec-pd-client"), codec}, nil
```

Comment:
Should we also extract this as a constant to reuse it in line 61?

Reply:
LGTM